base/int.jl

```diff
@@ -553,14 +553,75 @@ floor(::Type{T}, x::Integer) where {T<:Integer} = convert(T, x)
 
 ## integer construction ##
 
+"""
+    @int128_str str
+
+Parse `str` as an [`Int128`](@ref).
+
+# Examples
+```jldoctest; filter = r"Stacktrace:(\\n \\[[0-9]+\\].*)*"
+julia> @int128_str "123456789123"
```

Comment:
Is this form even worth showing?

Reply:
Same with the other once.

Reply:
I think it is worth showing just because the doc string has that format. It's not very long...